packages/@react-types/combobox/src/index.d.ts

```diff
@@ -10,9 +10,10 @@
  * governing permissions and limitations under the License.
  */
 
-import {CollectionBase, DOMProps, FocusableProps, InputBase, SingleSelection, SpectrumLabelableProps, StyleProps, TextInputBase, Validation} from '@react-types/shared';
+import {AsyncLoadable, CollectionBase, DOMProps, FocusableProps, InputBase, SingleSelection, SpectrumLabelableProps, StyleProps, TextInputBase, Validation} from '@react-types/shared';
 
-export interface ComboBoxProps<T> extends CollectionBase<T>, SingleSelection, InputBase, TextInputBase, DOMProps, Validation, FocusableProps {
+// TODO: Should AsyncLoadable go in ComboBoxProps or SpectrumComboBoxProps
```

Comment:
Open question, I figure async loading isn't spectrum specific so I put it here

Reply:
Interesting question. There's nothing in react-aria or stately for it so I'd say it's spectrum specific at least for now.